src/main/java/com/glodblock/github/common/block/BlockFluidBuffer.java

```diff
@@ -48,7 +48,7 @@ public boolean onActivated(World world, int x, int y, int z, EntityPlayer player
             if (fs == null && ias != null) {
                 player.addChatMessage(
                         new ChatComponentText(
-                                Tooltip.fluidFormat(ias.getFluid().getLocalizedName(), ias.getStackSize())));
+                                Tooltip.fluidFormat(ias.getFluidStack().getLocalizedName(), ias.getStackSize())));
```

Comment:
Is there some benefit to calling this on the stack over the fluid? This looks like it may create new objects now (if not, disregard)

Reply:
`Fluid.getLocalizedName()` is deprecated and `FluidStack.getLocalizedName()` should be used (the methods may return different Strings). It does create a new FluidStack though, yeah.